src/reply/send_reply.rs

```diff
@@ -44,6 +44,18 @@ pub async fn say_reply<U, E>(
     send_reply(ctx, |m| m.content(text.into())).await
 }
 
+/// Shorthand of [`send_reply`] for text-only messages, and reply to the reference message for [`crate::PrefixContext`]
+///
+/// for [`crate::ApplicationContext`], this is the same as [`say_reply`].
+///
+/// Note: panics when called in an autocomplete context!
+pub async fn reply_reply<U, E>(
```

Comment:
I say remove `reply_reply` - those plain functions for replying have become kinda useless anyways (`say_reply(ctx, "...")` is inferior to `ctx.say("...")`) so it's not really important to add this new utility feature to them

Reply:
do you mean I should directly use `send_reply()` builder in `ctx.reply()` ?

Reply:
Yes, or even `ctx.send(|b| b.content(text).reply(true))`